WordPress/Classes/ViewRelated/Notifications/Controllers/NotificationsViewController/NotificationsViewController.swift

```diff
@@ -845,26 +836,47 @@ extension NotificationsViewController {
             guard let self = self else {
                 return
             }
-
             self.view.isUserInteractionEnabled = true
-
+            let viewController: UIViewController?
             if note.kind == .comment {
-                guard let commentDetailViewController = self.notificationCommentDetailCoordinator.createViewController(with: note) else {
-                    DDLogError("Notifications: failed creating Comment Detail view.")
-                    return
-                }
+                viewController = getNotificationCommentDetailViewController(for: note)
+            } else {
+                viewController = getNotificationDetailsViewController(for: note)
+            }
+            guard let viewController = viewController else { return }
+            displayViewController(viewController)
```

Comment:
[Nit] Guard doesn't read too well at the end of a function with only one line after it. That's my personal opinion though, no strict rule for that.
```suggestion
            if let viewController {
                displayViewController(viewController)
            }
```